### PR TITLE
Expose ps cluster

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Review by spron-in — APPROVED** [review]
please confirm ports with @egegunes

File: source/expose.rst [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+applications in several scenarios. In either way the cluster is exposed with
+regular Kubernetes `Service objects<https://kubernetes.io/docs/concepts/services-networking/service/>`_,
+configured by the Operator.
+
+This document describes the usage of :ref:`Custom Resource manifest options<operator.custom-resource-options>`
+to expose the clusters deployed with the Operator. The expose options vary for
+different replication types: `Asyncronous<https://dev.mysql.com/doc/refman/8.0/en/replication.html>`_
+and `Group Replicaiton <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_.
+
+Asyncronous Replication
```
Comment (egegunes) [inline source/expose.rst:14]: saw the same typo on multiple places

```suggestion
Asynchronous Replication
```

File: source/expose.rst [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+-----------------------
+
+With `Asyncronous or Semi-syncronous replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication-primary-secondary-replication.html>`_
```
Comment (egegunes) [inline source/expose.rst:17]:
```suggestion
With `Asynchronous or Semi-synchronous replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication-primary-secondary-replication.html>`_
```

File: source/expose.rst [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+================
+
+The Operator provides entry points for accessing the database by client
+applications in several scenarios. In either way the cluster is exposed with
+regular Kubernetes `Service objects<https://kubernetes.io/docs/concepts/services-networking/service/>`_,
+configured by the Operator.
+
+This document describes the usage of :ref:`Custom Resource manifest options<operator.custom-resource-options>`
+to expose the clusters deployed with the Operator. The expose options vary for
+different replication types: `Asyncronous<https://dev.mysql.com/doc/refman/8.0/en/replication.html>`_
```
Comment (egegunes) [inline source/expose.rst:11]:
```suggestion
different replication types: `Asynchronous <https://dev.mysql.com/doc/refman/8.0/en/replication.html>`_
```

File: source/expose.rst [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+and `Group Replicaiton <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_.
```
Comment (egegunes) [inline source/expose.rst:12]:
```suggestion
and `Group Replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_.
```

File: source/expose.rst [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+``<CLUSTER_NAME>-mysql-primary``: for example, ``cluster1-mysql-primary``.
+
+.. image:: ./assets/images/exposure-async.svg
+   :align: center
+
+This Service is created by default and is always present. You can change the
+type of the Service object by setting :ref:`mysql.primaryServiceType<mysql-primaryservicetype>`
+variable in the Custom Resource.
+
+The following example exposes the Primary node of the asyncronous cluster with
```
Comment (egegunes) [inline source/expose.rst:28]:
```suggestion
The following example exposes the Primary node of the asynchronous cluster with
```

**Review by egegunes — CHANGES_REQUESTED** [review]

**Pull request merged**